lib/generator.dart

```diff
@@ -510,18 +511,19 @@ class _GeneratorVisitor extends RecursiveVisitor {
           leafType.name.value == scalar.graphQLType) {
         final graphqlTypeSafeStr = TypeName(
             name: gql
-                .buildTypeString(node.type, context.options,
+                .buildTypeName(node.type, context.options,
                     dartType: false, schema: context.schema)
+                .namePrintable
                 .replaceAll(RegExp(r'[<>]'), ''));
-        final dartTypeSafeStr =
-            TypeName(name: dartTypeStr.replaceAll(RegExp(r'[<>]'), ''));
+        final dartTypeSafeStr = TypeName(
+            name: dartTypeName.namePrintable.replaceAll(RegExp(r'[<>]'), ''));
```

Comment:
`replaceAll(RegExp(r'[<>]'), '')` could be moved into the main `Name` class. to gather all the naming juggling in one place)

```
String get dartTypeSafe => namePrintable.replaceAll(RegExp(r'[<>]'), '');
```